### ISSUE TITLE
Frozen Python Interpreter Missing in 2.0

### ISSUE THREAD

**Issue by gregwalters** [NagiosEnterprises/ncpa#288]
First off, jomann09, I appreciate all the work you're putting into maintaining this project. NCPA is critical in my org and we couldn't do it without you.

After reading the comments in #68 I've been chomping at the bit to use the built in interpreter that I assumed would come with the 2.0 release and it doesn't seem like it's there:

`[root@######### ncpa]# pwd
/usr/local/ncpa
[root@######### ncpa]# find . -iname "*python*"
./libpython2.7.so.1.0
[root@######### ncpa]# ls /usr/local/ncpa/Python
ls: cannot access /usr/local/ncpa/Python: No such file or directory
[root@######### ncpa]# rpm -qa | grep -i ncpa
ncpa-2.0.0-1.el7.centos.x86_64`

Did I miss it somewhere? Is there another way to access the bundled python?

**Comment by jomann09** [comment]
No, it's not actually bundled in there anymore. I can look into changes that have been made that might have caused it to not be added anymore. I haven't seen it since I posted that comment in that message.

**Comment by jomann09** [comment]
I actually also just installed NCPA 1.7.x and 1.8.x on EL6 and Windows and I didn't see the binary file... starting to wonder what I was doing that I saw the Python binary.

**Comment by gregwalters** [comment]
Like #68, the reason I've been interested in accessing the built-in interpreter is to run plugins. In situations where installing python isn't an option (change-controlled Windows servers) it'd be awesome to write one cross-platform plugin that could get deployed everywhere versus maintaining a distinction between PowerShell plugins and python plugins with essentially the same functionality.  

Heck, given access to a bundled python I'd use NCPA plugins to manage other NCPA plugins! Example insecure POC: https://gist.github.com/gregwalters/235b804d26de62a0033e06309051b8a5

**Comment by jomann09** [comment]
I can see a lot use in it if it were possible, I was going to actually write about it in the new docs until I couldn't find it again. I've been testing some cx_Freeze settings to try to get it to include it so I will continue trying to add it into the build.

**Comment by jomann09** [comment]
I think I was building for Mac at the time. It looks like the Mac OS X version comes with a "Python" binary but it isn't an actual interpreter just the library - or so it seems. So basically it looks like we would need a way to run scripts via an executable that would run them on the frozen python executable.

**Comment by jomann09** [comment]
I've added this into NCPA 3 - I am actually just copying over the executable for the interpreter.
5408ed7
Currently works in Windows. I still have to test this in Mac OS X and Linux though.

**Comment by gregwalters** [comment]
Awesome. Looking forward to the 3.0 release :)

**Comment by jomann09** [comment]
I will definitely be building some builds of 3 that people can play around with. I've just now been testing the new changes on Linux and Mac OS X

**Comment by gregwalters** [comment]
Let me know when it's build for Linux and I'll test.
